MultiSource/Benchmarks/Ptrdist/ks/KS-2.c

```diff
@@ -339,16 +341,17 @@ main(int argc, char **argv)
     }
 
     /* prepare the data structures */
-    ReadNetList(argv[1]);
-    NetsToModules();
-    ComputeNetCosts();
+    _Checked {ReadNetList(argv[1]);}
+    _Checked {NetsToModules();}
+    _Checked {ComputeNetCosts();}
```

Comment:
I would have a single `_Checked` block around all 3 calls, fewer changes that way. It could even extend all the way to line 403.